### PR TITLE
fix: Menu button widget height not changing on resizing

### LINKED ISSUE #25738: [Bug]: Unable to increase height of menu button widget
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Unable to increase height of menu button widget, The resize indicators show correctly but the widget's actual height doesn't reflect

### Steps To Reproduce

Try to increase height using resize indicators

Screenshot 2023-07-26 at 5 30 47 PM

### Public Sample App

_No response_

### Environment

Production

### Issue video log

_No response_

### Version

Cloud/Self Hosted

### PR DESCRIPTION
**Pull request by dhruvikn**
## Description
Allows the menu button's height to change as the widget is resized.

#### PR fixes following issue(s)
Fixes #25738

#### Type of change
- Bug fix (non-breaking change which fixes an issue)

- [x] Manual
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [x] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Review by jsartisan — APPROVED** [review]

File: app/client/src/widgets/MenuButtonWidget/component/index.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ const PopoverStyles = createGlobalStyle<{
     border-radius: 0;
   }
 
-  & > .${Classes.POPOVER2_TARGET} {
+  .${Classes.POPOVER2_TARGET} {
```
Comment (aswathkk) [inline app/client/src/widgets/MenuButtonWidget/component/index.tsx:62]: @dhruvikn Won't this set `height: 100%` all `.bp3-popover2-target` in the app?

Comment (dhruvikn) [reply app/client/src/widgets/MenuButtonWidget/component/index.tsx:62]: Ah yes, you're right. Thanks for bringing this up. I'll push a fix.

Comment (dhruvikn) [reply app/client/src/widgets/MenuButtonWidget/component/index.tsx]: Fixed. Please review.

**Review by jsartisan — APPROVED** [review]

**Review by aswathkk — APPROVED** [review]

**Comment by laveena-en** [comment]
Verified, PR looks good to me!

**Pull request merged**